db.js

```diff
@@ -372,12 +372,12 @@ var loans = (function () {
       return loans.baseQuery(`WHERE l.requester_id = '${userId}'`, userId);
     },
     getLoanById: function (loanId) {
-      return loans.baseQuery(`WHERE l.id = '${(loanId, 0)}'`);
+      return loans.baseQuery(`WHERE l.id = '${loanId}'`, 0);
```

Comment:
It isn't really clear here why you're passing in the value 0 because the `baseQuery` looks like it needs this value. Shouldn't you be passing in `req.session.userId` into this function from `loans.js`?

Reply:
Also on line 363, same thing